### PR TITLE
Feature/cu kd5r5u cannot integrate new aws platform by following aws documentation or executing terraform modules

### PR DESCRIPTION
**Pull request by pubudug**
[preview.pdf](https://github.com/meshcloud/meshcloud-docs/files/6739835/preview.pdf)

### CODE REVIEW

**Comment by meshcloud-gmbh** [comment]
Task linked: [CU-kd5r5u Cannot Integrate new AWS Platform by following AWS Documentation or executing Terraform Modules](https://app.clickup.com/t/kd5r5u)

File: docs/meshstack.aws.index.md [hunk 1]
```diff
@@ -30,380 +29,219 @@ When configuring these roles, operators must take care to correctly guard agains
...
-          "iam:UpdateSAMLProvider",
-          "iam:DeleteSAMLProvider",
-          "iam:ListSAMLProviders"
-        ],
-        "Resource": [
-          "arn:aws:iam::<ACCOUNT_ID>:saml-provider/*",
-          "arn:aws:cloudformation:*:<ACCOUNT_ID>:stack/meshstack-cf-access*"
-        ]
+        {
+            "Sid": "VisualEditor0",
```
Comment (sttomm) [inline docs/meshstack.aws.index.md:174]: d: This sid is looking a bit akward. Can we change it to something more meaningful? It's also set at other places. If you replace it, please replace it everywhere

**Review by sttomm — APPROVED** [review]
lgtm on top level. I am not that deep into it to understand whether everything mentioned here also works, but as you tested it with meshcloud prod, i think it should all be fine

File: docs/meshstack.aws.index.md [hunk 2]
```diff
@@ -15,13 +15,12 @@ This makes AWS available to meshProjects like any other cloud platform in meshSt
...
+meshStack uses [AWS Organizations](https://aws.amazon.com/organizations/) to provision and manage AWS Accounts for [meshProjects](./meshcloud.project.md). To use AWS with a meshStack deployment, operators will need an AWS [management account](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_getting-started_concepts.html) acting as the parent of all accounts managed by meshStack. The complete meshStack setup contains three dedicated accounts:
 
-* meshcloud Account - Here lives the `meshfed-service` user.
-* Org Root Account - All accounts created by meshStack are living "under" this account and its Organization Units. `meshfed-service` needs to assume a role in this account.
+* meshCloud Account - `meshfed-service-user` should be created in this account. We have a dedicated account for this user so that meshCloud can easily roll the credentials of the user when needed.
```
[the remark this answers is not in the document]
Comment (pubudug) [reply docs/meshstack.aws.index.md:20]: ah Okay. Will change it.

File: docs/meshstack.aws.index.md (hunk 1 above)
Comment (pubudug) [reply docs/meshstack.aws.index.md:174]: Will change.

**Comment by pubudug** [comment]
Merging this because other tickets are also being implemented in the same area and would be good to have these changes in. @chewbeccy is on vacation and @Jelledb & @tfelix is on meshTraction. If any changes are needed we do it later I guess.

**Pull request merged**